csharp/language-reference/operators/UserDefinedConversions.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+public struct Digit
```

Comment:
This could be a `readonly struct`, although that’s not what this article is discussing. 

What are your thoughts? I’m not sure I’d propose general guidance to create `readonly struct` types whenever possible, but it might be the preferred choice when a struct is designed to be immutable.

Reply:
@BillWagner I think it would be a good addition to raise awareness about `readonly` structs. Also I've noticed that in the corefx repo (and most probably in the coreclr as well then) as many as possible structs have been marked as `readonly`. So, marking applicable struct examples in the docs as `readonly` follows the trend. I've marked both `Digit` and `Fraction` structs as `readonly`. Thanks for the notice!